src/main/java/net/minecraftforge/event/world/GetCollisionBoxesEvent.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.event.world;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.world.World;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.fml.common.eventhandler.Cancelable;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+/**
+ * This event is fired during {@link World#collidesWithAnyBlock(AxisAlignedBB)}
+ * and before returning the list in {@link World#getCollisionBoxes(Entity, AxisAlignedBB)}<br>
+ * <br>
+ * {@link #entity} contains the entity passed in the {@link World#getCollisionBoxes(Entity, AxisAlignedBB)}. <b>Can be null.</b> Calls from {@link World#collidesWithAnyBlock(AxisAlignedBB)} will be null.<br>
+ * {@link #aabb} contains the AxisAlignedBB passed in the method. <b>Do not modify this AABB</b><br>
```

Comment:
Oh huh, I guess this became a thing when they stopped pulling from a pooled resource? Good to know, will fix.